src/Adapter/MSTest.TestAdapter/Discovery/AssemblyEnumerator.cs

```diff
@@ -302,6 +302,11 @@ private static bool TryProcessTestDataSourceTests(UnitTestElement test, TestMeth
             try
             {
                 data = dataSource.GetData(methodInfo);
+
+                if (!data.Any())
```

Comment:
Could you please add a comment to mention the other place to look at/change? And mention this one on the `TstMethodRunner`.

Reply:
Added,